.circleci/config.yml

```diff
@@ -4,7 +4,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/golang:1.9.2-stretch
+      - image: circleci/golang:1.11-stretch
```

Comment:
Are you sure you don't want to wait until 1.12?

https://github.com/golang/go/milestone/65

Reply:
I think we should do that in a separate PR when it's released.  The 1.11 series has been in the wild for many months already.